### DISCUSSION TITLE
I can't download the videos after the last update

### DISCUSSION THREAD

**Discussion by Pedro-dos-Santos-Junior** [aclap-dev/video-downloadhelper#554] [Help and Q&A]
### Your question

I can't download the files of a video course, I paid for the extension just for that, after the last update I can't download anything else from the course.

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://ead.institutokyoto.com.br/31236-japones-completo-do-zero-ao-avancado/720035-boas-vindas-e-instrucoes",
  "error": {
    "details": "SideDownload error: : [hls @ 000001aa25d76940] Error when loading first segment 'https://cdn.vod.br1.jmvstream.com/vod/vod_9677/f/7reh381f80hnmer/h/4/1080/chunck_list_JMV0.ts'\r\n[in#0 @ 000001aa25d76780] Error opening input: Invalid data found when processing input\r\nError opening input file https://cdn.vod.br1.jmvstream.com/vod/vod_9677/f/7reh381f80hnmer/h/4/1080/chunck_list_JMV.m3u8?uf085lum5fbz=f6lv79zo5tx1.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_6168865536018137",
    "error": "coapp_failure",
    "id": "coapp_failure_f3a8bee5-610e-48a7-9f04-2f49b6ed5134"
  }
}

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://ead.institutokyoto.com.br/31236-japones-completo-do-zero-ao-avancado/720035-boas-vindas-e-instrucoes",
  "error": {
    "details": "SideDownload error: : [hls @ 000001aa25d76940] Error when loading first segment 'https://cdn.vod.br1.jmvstream.com/vod/vod_9677/f/7reh381f80hnmer/h/4/1080/chunck_list_JMV0.ts'\r\n[in#0 @ 000001aa25d76780] Error opening input: Invalid data found when processing input\r\nError opening input file https://cdn.vod.br1.jmvstream.com/vod/vod_9677/f/7reh381f80hnmer/h/4/1080/chunck_list_JMV.m3u8?uf085lum5fbz=f6lv79zo5tx1.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_6168865536018137",
    "error": "coapp_failure",
    "id": "coapp_failure_f3a8bee5-610e-48a7-9f04-2f49b6ed5134"
  }
}

**Comment by Mattmjs** [comment]
Is this a Japanese anime your trying to download ? It does not look like it works in the extension, not even ffmpeg on it's own.
I got something that was only a few seconds long in another tool in **yt-dlp**.

**Reply by Pedro-dos-Santos-Junior** [reply]
This is a  course to learn Japanese, i wanted to download the videos because the course is running out of time and I have to buy it again
